index.js

```diff
@@ -12,20 +12,19 @@ function getHeaders(method, resource, payload) {
   };
 }
 
-function getQs(appId, pagination) {
+function getQs(appId, params = {}) {
```

Comment:
Not sure if node 6.10 (latest on Lambda where this client is often used) supports default params.